arch/arc/core/mpu/arc_mpu_v6_internal.h

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (c) 2021 Synopsys.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_ARCH_ARC_CORE_MPU_ARC_MPU_V6_INTERNAL_H_
+#define ZEPHYR_ARCH_ARC_CORE_MPU_ARC_MPU_V6_INTERNAL_H_
+
+#define AUX_MPU_RDB_VALID_MASK (0x1)
+#define AUX_MPU_EN_ENABLE   (0x40000000)
+#define AUX_MPU_EN_DISABLE  (0xBFFFFFFF)
+#define AUX_MPU_EN_BANK_MASK (0x1)
+
+#define AUX_MPU_RDP_REGION_SIZE(bits)   (((bits - 1) & 0x3) | (((bits - 1) & 0x1C) << 7))
+
+#define AUX_MPU_RDP_ATTR_MASK (0x1F8)
```

Comment:
Frankly I don't like these cryptic numbers. I guess these are some groups of bits shifted left?
Look at `BIT_MASK()` in https://github.com/zephyrproject-rtos/zephyr/blob/main/include/sys/util_macro.h#L68. Then we'll get something like `BIT_MASK(xxx) << yyy` which is much more readable.